test/test_nngraph.lua

```diff
@@ -343,7 +343,7 @@ function test.test_annotateGraph()
   local function checkDotFile(tmpfile)
     local dotcontent = io.open(tmpfile .. '.dot', 'r'):read("*all")
     tester:assert(
-        dotcontent:match('%[label=%"Input.*DescA.*%" color=red%]'))
+        dotcontent:match('%[label=%"Input.*DescA.*%" color=red.*%]'))
```

Comment:
This is necessary for this test to pass with our internal fork of the project. This is a tiny change and I think it improves the test, but I'm happy to remove it if you prefer.